projects/vdk-core/src/vdk/internal/builtin_plugins/job_properties/datajobs_service_properties.py

```diff
@@ -1,17 +1,21 @@
 # Copyright 2021 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
 import logging
+from copy import deepcopy
 from typing import Dict
+from typing import List
+from typing import Optional
 from typing import Union
 
 from vdk.api.job_input import IProperties
 from vdk.api.plugin.plugin_input import IPropertiesServiceClient
 
+from ...core.errors import log_and_throw
+from ...core.errors import ResolvableBy
```

Comment:
Why dots `...` instead of full path?

Reply:
also spotted that, it's due IDE auto-import; didn't deep-dive into details and presumed it's some syntax sugar.
Do you have something in mind?

Reply:
in case we introduce platform-level properties/ingestion preprocessors (excluding vdk-test-utils for test purposes), 
we may add in both - an additional except cases for PlatformError raised to simply rethrow it

Reply:
Not sure if the last comment is for this thread.

Reply:
The dots indicate relative import. 

Per PEP 8 (https://peps.python.org/pep-0008/) Absolute imports are recommended over relative imports.